### PR TITLE
[vRO] support

### PR DESCRIPTION
**Pull request by ya4ept**
- added sync packages (from twRO)
- added actor_action, actor_info_request, actor_name_request, map_login, skill_use, skill_use_location

### CODE REVIEW

**Comment by chumlacklx** [comment]
image
Diconnect occur oftenly may be due to sever had new update this morning.

**Review by alisonrag — APPROVED** [review]

File: tables/vRO/recvpackets.txt [hunk 1]
```diff
@@ -1258,14 +1255,14 @@
 0B0F -1 4 1
 0B10 10 10 1
 0B11 4 4 1
-0B12 2 2 -1
+0B12 2 2 4294967295
```
Comment (sctnightcore) [inline tables/vRO/recvpackets.txt:1258]: -1

**Review by sctnightcore — COMMENTED** [review]
TODO

**Comment by royalgarter** [comment]
There's a bug in this commit. Several reports that bot could not move at all and always "Stuck during route."

**Comment by ya4ept** [comment]
> There's a bug in this commit. Several reports that bot could not move at all and always "Stuck during route."

I think this does not apply to this commit

**Comment by royalgarter** [comment]
> I think this does not apply to this commit

Ah sorry, but it's fixed with [last merged for vRO-support #e40395a] btw.(https://github.com/OpenKore/openkore/commit/e40395ab151cc223f30daea3fe380852e4ec36dc)

**Review by alisonrag — APPROVED** [review]

File: src/Misc.pm [hunk 2]
```diff
@@ -3937,22 +3937,23 @@ sub useTeleport {
...
-			unless ($item) { $item = $char->inventory->getByNameID(12323); } # Novice Fly Wing
-			unless ($item) { $item = $char->inventory->getByNameID(23280); } # Beginner's Fly Wing
-		} else {
+		if ($config{teleportAuto_item1}) {
 			$item = $char->inventory->getByName($config{teleportAuto_item1});
 			$item = $char->inventory->getByNameID($config{teleportAuto_item1}) if (!($item) && $config{teleportAuto_item1} =~ /^\d{3,}$/);
+		} unless ($item) {
+			$item = $char->inventory->getByNameID(23280); # Beginner's Fly Wing
+			unless ($item) { $item = $char->inventory->getByNameID(12323); } # Novice Fly Wing
+			unless ($item) { $item = $char->inventory->getByNameID(601); } # Fly Wing
```
Comment (ya4ept) [inline src/Misc.pm:3946]: what about
```perl 
if ($config{teleportAuto_item1}) {
	$item = $char->inventory->getByName($config{teleportAuto_item1});
	$item = $char->inventory->getByNameID($config{teleportAuto_item1}) if (!($item) && $config{teleportAuto_item1} =~ /^\d{3,}$/);
} 
$item = $char->inventory->getByNameID(23280) unless $item; # Beginner's Fly Wing
$item = $char->inventory->getByNameID(12323) unless $item; # Novice Fly Wing
$item = $char->inventory->getByNameID(601) unless $item;     # Fly Wing
```

**Comment by abc9900** [comment]
Can you help me to fix this error? i can't find solution anywhere...

File: src/Misc.pm (hunk 2 above)
Comment (ya4ept) [reply src/Misc.pm:3946]: @alisonrag

Comment (alisonrag) [reply src/Misc.pm:3946]: I will fix in some hours :)

**Comment by alisonrag** [comment]
> Can you help me to fix this error? i can't find solution anywhere...
> 

https://discord.gg/N6ghKazB

File: src/Misc.pm (hunk 2 above)
Comment (alisonrag) [reply src/Misc.pm]: fixed at:
https://github.com/OpenKore/openkore/pull/3460/commits/14b3ae12107a53cd2ff329c698d301cb3e83bfdc

**Comment by ya4ept** [comment]
@alisonrag @sctnightcore, let's merge it?

**Comment by alisonrag** [comment]
seems ok to me

**Comment by alisonrag** [comment]
just need tp be careful about poseidon changes

**Pull request merged**